tsconfig.base.json

```
@@ -582,6 +582,9 @@
       "@island.is/next-ids-auth": ["libs/next-ids-auth/src/index.ts"],
       "@island.is/nova-sms": ["libs/nova-sms/src/index.ts"],
       "@island.is/plausible": ["libs/plausible/src/index.ts"],
+      "@island.is/portals-admin-icelandic-names-registry": [
```

NX now defaults to this type of import path but we should keep it consistent with existing import paths. You'll have to find and replace this in import statements (not workspace.json).

```suggestion
      "@island.is/portals/admin/icelandic-names-registry": [
```